src/test/java/com/icthh/xm/tmf/ms/communication/config/SmppTest.java

```diff
@@ -54,13 +54,4 @@ public void testSendingSms() {
         assertNotNull(messageId);
     }
 
-    @Test
```

Comment:
Why test removed?

Reply:
coz i have removed the method this test tested